modules/compliance-apply-remediation-for-customized-mcp.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: PROCEDURE
+[id="complianc-operator-apply-remediation-for-customized-mcp"]
+= Applying remediation when using customized machine config pools
+
+When you create a custom `MachineConfigPool`, add a label to the `MachineConfigPool`.
```

Comment:
@xenolinux Hi, I think we should add complete procedure to create custom machineConfigPool.

1. Label node 
Example: 
> $ oc get nodes
> NAME                                         STATUS   ROLES    AGE     VERSION
> ip-10-0-128-92.us-east-2.compute.internal    Ready    master   5h21m   v1.23.3+d99c04f
> ip-10-0-158-32.us-east-2.compute.internal    Ready    worker   5h17m   v1.23.3+d99c04f
> ip-10-0-166-81.us-east-2.compute.internal    Ready    worker   5h17m   v1.23.3+d99c04f
> ip-10-0-171-170.us-east-2.compute.internal   Ready    master   5h21m   v1.23.3+d99c04f
> ip-10-0-197-35.us-east-2.compute.internal    Ready    master   5h22m   v1.23.3+d99c04f
> 
> $ oc label node ip-10-0-166-81.us-east-2.compute.internal node-role.kubernetes.io/wscan=
> node/ip-10-0-166-81.us-east-2.compute.internal labeled

2. Create custom machineConfigPool
If possible then please add complete mcp definition like below 

> apiVersion: machineconfiguration.openshift.io/v1
> kind: MachineConfigPool
> metadata:
>   name: wscan
>   labels:
>     pools.operator.machineconfiguration.openshift.io/wscan: ''
> spec:
>   machineConfigSelector:
>     matchExpressions:
>       - {key: machineconfiguration.openshift.io/role, operator: In, values: [worker,wscan]}
>   nodeSelector:
>     matchLabels:
>       node-role.kubernetes.io/wscan: ""

3. verify mcp is created successfully

> $ oc get mcp -w

Reply:
@pdhamdhe Addressed the changes